package-lock.json

```diff
@@ -37564,7 +37564,6 @@
       "integrity": "sha512-8q67ORQ9O0Ms0nlqsXTVhaBefRBaLrzPxOewAZhdcO7onHwcO5/wRdWtHhZgfpCZlhY7NogkU16z3WnorSSkEA==",
       "dev": true,
       "requires": {
-        "@babel/core": "^7.11.0",
```

Comment:
I'm confused about the changes here (we didn't touch `package.json`, but I guess that's okay?

Reply:
These changes have been hanging around the last few days from an older merge. I usually exclude them from my commits, but this time it snuck through.